src/App.jsx

```diff
@@ -1,7 +1,53 @@
-import data from "./data.json";
+import { Album } from "./components/Album"
+import data from "./data.json"
 
-console.log(data);
+const selectAlbums = data.albums.items
 
-export const App = () => {
-  return <div>Find me in src/app.jsx!</div>;
+export const App = () =>{
+return(
+  <div>
+      <div className="albums">
+        {selectAlbums.map(album => (
+          <Album 
+            key={album.id}
+            artistName={album.artists[0].name}
+            artistUrl={album.artists[0].external_urls.spotify}
+            coverImage={album.images[0].url}
+            albumName={album.name}
+            albumUrl={album.external_urls.spotify}
+          />
+        ))}
+      </div>
+      <footer>
+        <p>2024 by Arnau Vidal</p>
+        <img src="src/assets/icons/technigo-logo.svg" alt="Technigo" className="logo" />
```

Comment:
You're already in the src so try with "./assets/icons/technigo-logo.svg"

Reply:
I copied the relative path directly from vsc side bar and I got that and seems like it's working too "src\assets\icons\technigo-logo.svg". Any opinions on that?

Reply:
using "npm run dev" the logo looks ok with this path:
 ./src/assets/icons/technigo-logo.svg
 
 It isn't working when pushed into github in the browser, I don't understand what's wrong.